composer.json

```
@@ -4,15 +4,15 @@
     "type": "project",
     "license": "MIT",
     "require": {
-        "php": ">=7.2.5",
+        "php": ">=8.0.2",
         "ext-ctype": "*",
         "ext-iconv": "*",
-        "symfony/console": "5.*",
-        "symfony/dotenv": "5.*",
+        "symfony/console": "6.*",
+        "symfony/dotenv": "6.*",
         "symfony/flex": "^1.17",
-        "symfony/framework-bundle": "5.*",
+        "symfony/framework-bundle": "6.*",
         "symfony/runtime": "*",
```

@yceruto Is this correct?
Shouldn't it be:

```suggestion
        "symfony/runtime": "6.*",
```